docs/Project-Configuration/Project-Commands.md

```diff
@@ -760,13 +760,196 @@ information is not necessary.
 
 Parameters:
 
--   `file`: the JSON or YAML file containing the test results, see
-    <https://github.com/evergreen-ci/poplar> for more info.
--   `aws_key`: your AWS key (use expansions to keep this a secret)
--   `aws_secret`: your AWS secret (use expansions to keep this a secret)
--   `region`: AWS region of the bucket, defaults to us-east-1.
--   `bucket`: the S3 bucket to use.
--   `prefix`: prefix, if any, within the s3 bucket.
+| Name         | Type   | Description                                                                         |
```

Comment:
I would keep the command parameters as they were (following the convention of the rest of the file). The table is fine for the file schema definition.

Reply:
I agree. Done!